ci/environment-latest.yaml

```diff
@@ -0,0 +1 @@
+environment-3.8.yaml
```

Comment:
Does this work for you? I didn't think you could refer to other files from within an environment.yaml file.